specification/redis/resource-manager/Microsoft.Cache/stable/2023-04-01/redis.json

```diff
@@ -1964,6 +1964,10 @@
             "authnotrequired": {
               "type": "string",
               "description": "Specifies whether the authentication is disabled. Setting this property is highly discouraged from security point of view."
+            },
+            "storage-subscription-id": {
+              "type": "string",
+              "description": "SubscriptionId of the storage account for persistence (aof/rdb) using ManagedIdentity. Default value is subscriptionId of the cache."
```

Comment:
I have a doubt about whether it makes sense to say that the default value is the subscription id of the cache.  Because that would _imply_ that on a GET request, the subscription id of the cache will come back, if the storage subscription id wasn't already specified by a PUT with this new api version...

AND that wouldn't seem intuitive for that subscription id to be claimed to be the cache's subscription if the storage account is NOT really in the cache's subscription.

(In other words, if someone provided a storage account earlier, and no subscription id (or configured it with an old api version), can you know that it is really in the subscription id of the cache, and that that is a sensible default, for you to return, when someone does a later request with this api-version?)

Reply:
Good point, i can definitely see the implication of mentioning default value. 
I was thinking about rephrasing with a mention of "when subscription id is different from cache subscription id" but that wouldn't complete since it excludes scenario of passing cache subscriptionId in this parameter.
I have removed the mention of default value.